examples/NIH-example.md

```diff
@@ -2,16 +2,21 @@
 
 This resource sharing plan was written by Daniel Himmelstein and Casey Greene in December 2016 for an NIH grant proposal.
 The grant's field was biomedical informatics (Grant Number HG009516).
+This example was revised in April 2018 based on [community feedback](https://github.com/lab-carpentry/blueprint-resourcesharing/issues/4).
 
 This plan largely focuses on ensuring the open licensing of project outputs.
 Carly Strasser — a Program Officer for the [Data-Driven Discovery Initiative](https://www.moore.org/initiative-strategy-detail?initiativeId=data-driven-discovery) at the Gordon & Betty Moore Foundation — [suggests](https://github.com/lab-carpentry/blueprint-resourcesharing/pull/1#issuecomment-266153553) adding more details regarding your specific data outputs and corresponding sharing plans.
 For inspiration, see the [Digital Curation Centre guide](http://www.dcc.ac.uk/resources/how-guides/develop-data-plan "How to Develop a Data Management and Sharing Plan") or [public Data Management Plans](https://dmptool.org/public_dmps "DMPTool").
 
+In addition, we recommend exercising caution when combining this sharing plan with boilerplate text provided by your institution.
+In [some cases](https://github.com/lab-carpentry/blueprint-resourcesharing/issues/4#issuecomment-379023061), universities include text declaring their intention to exclusively control intellectual property resulting from the award.
+While this sharing plan likely takes precedence due to its specificity, it's message may be diluted if combined with a defense of proprietary control of publicly-funded outputs.
+
 ***
 
 ## Resource Sharing Plan
 
-All outputs from this project — including code, data, figures, documentation, and manuscripts — will be made publicly available under an open license within two years of the end of the award.
+All outputs from this project — including code, data, figures, documentation, and manuscripts — will be made publicly available under an open license as soon as a stable version is available.
```

Comment:
Should this say:
> All outputs from this project — including code, data, figures, and documentation— will be made publicly available under an open license as soon as a stable version is available or an associated manuscript is published, whichever is earlier.

Reply:
I realize that what you have is a bit stronger b/c the manuscripts also fall under this, but it appears that this was confusing to a couple of the reviewers that have seen grants discussed in #4.

Reply:
I think manuscripts would probably fall under "Figures, documentation, and writing"? Do we want to do "Figures, documentation, and writing (including manuscripts)"?

I think preprints should be mandatory CC BY, but it's unclear whether our policy would allow submitting to non-open journals like the PSB proceedings which uses CC BY-NC.

Reply:
I would be in favor of adding a bit about how preprints will be uploaded under an open license. I don't think including manuscripts in that is necessarily helpful, because people might not realize that some journals do not allow an open license.